README.md

```diff
@@ -155,14 +155,18 @@ Be sure to call `validate()` on the `changeset` before saving or committing chan
 
 ## Types
 
-Do you consume `ember-changeset` and need to type an object?
-
-
 ```ts
 import { BufferedChangeset } from 'ember-changeset/types';
+import { Changeset } from 'ember-changeset';
 
 export default class Foo extends Component {
   changeset?: BufferedChangeset;
+
+  constructor() {
+    super(...arguments);
+
+    this.changeset = new Changeset(obj);
+  }
```

Comment:
This example needs a couple tweaks!

1. The `...arguments` approach, while the usual for plain JS, will actually not currently type-check.
2. The `changeset` should not be optional, as it's set up in the constructor.
3. Where does `obj` come from? I'd assume it's an argument?

```ts
import Component from '@glimmer/component';
import { BufferedChangeset } from 'ember-changeset/types';
import { Changeset } from 'ember-changeset';

interface Args {
  user: {
    name: string;
    age: number;
  }
}

export default class Foo extends Component<Args> {
  changeset: BufferedChangeset;

  constructor(owner, args) {
    super(owner, args);
    this.changeset = new Changeset(args.user);
  }
}
```

Something like this would be a better example, which would also fully type-check.